src/style.css

```
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin: 0 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  margin: 0 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  margin: 0 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  margin: 0 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  margin: 0 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  margin: 0 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

1) you have most of styles at the common class selector stars__star
2) you can list as a group
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(../src/images/star-active.svg);
}
```